src/surface.rs

```diff
@@ -47,6 +47,61 @@ impl Surface {
                                                                   height))
         }
     }
+
+    pub fn get_mime_data(&self, mime_type: &str) -> Option<&[u8]> {
```

Comment:
This is unsafe: nothing is preventing the caller to set a different data for the mimetype after calling this function, and then using the (now freed) byte array of the previously set data

Reply:
Should i make a copy of the data and return a vector?

Reply:
That would be one option, yes. Alternative would be something like the data getter from the image surface but that's very limiting (requires refcount of 1).

Reply:
What would be the advantage of using a getter similar to the that one in image surface?

Reply:
No copying of the data into a `Vec` :)

Reply:
Oh I thought something more than that. I used to_vec()

Reply:
Great, thanks :)